src/openrct2/CmdlineSprite.cpp

```diff
@@ -91,7 +91,7 @@ static bool sprite_file_open(const utf8* path)
     if (spriteFileHeader.num_entries > 0)
     {
         int32_t openEntryTableSize = spriteFileHeader.num_entries * sizeof(rct_g1_element_32bit);
-        rct_g1_element_32bit* openElements = static_cast<rct_g1_element_32bit*>(malloc(openEntryTableSize));
+        rct_g1_element_32bit* openElements = new rct_g1_element_32bit[openEntryTableSize];
```

Comment:
This can easily be replaced with a `std::unique_ptr`. To pass the ptr to fread use `.get()`